src/etwlogger_driver.c

```diff
@@ -219,7 +87,7 @@ static void perform_EventWriteLogErrorEvent(const char* content, const char* fil
 static void perform_EventWriteLogWarningEvent(const char* content, const char* file, const char* func, int line)
 {
     TraceLoggingWrite(g_hMyComponentProvider,
-        "LogError",
+        "LogWarning",
```

Comment:
>LogWarning [](start = 9, length = 10)

ha good catch

Reply:
:)

---
In reply to: [453753003](https://github.com/Azure/azure-c-logging/pull/11#discussion_r453753003) [](ancestors = 453753003)